### PR TITLE
Update careers page.

### PR DESCRIPTION
**Pull request by josh-padnick**
This PR:

- Cleans up the text of the [careers page](https://gruntwork.io/careers/).
- Adds a tiny amount of jQuery, which required add optional page-level script includes to ensure the jQuery code was executed _after_ the jQuery library was included in the page.

The changes have been pushed live.

### CODE REVIEW

File: pages/careers/_scripts.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    })(jQuery);
+
+    $().addScrollForCitation(1);
+    $().addScrollForCitation(2);
+    $().addScrollForCitation(3);
+    $().addScrollForCitation(4);
+    $().addScrollForCitation(5);
+
+  });
+</script>
```
Comment (brikis98) [inline pages/careers/_scripts.html]: We already have scrolling functionality: https://github.com/gruntwork-io/gruntwork-io.github.io/blob/master/assets/js/main.js#L36. All you have to add is a link to an anchor tag (`<a href="#foo">...`) and it'll add smooth scrolling automatically. I'd recommend removing the new scripts you added.

File: pages/careers/_open-positions.html [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
-    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're looking for:</p>
+    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're currently looking for:</p>
     <h3>Senior Full-Stack Engineer</h3>
+    <h4>What You'll Do</h4>
+    <ul>
+      <li><strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on Serverless apps in Node, write Terraform modules, write Python scripts and Bash scripts, do frontend development, learn the newest tech from AWS, invent new ways of managing infrastructure. We're looking for true full-stack engineers who can go deep in any area. We're less concerned with prior experience than we are with demonstrated ability to go deep when necessary.</li>
```
Comment (brikis98) [inline pages/careers/_open-positions.html]: Nice addition.

File: pages/careers/_open-positions.html [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>Demonstrated experience building good software.</li>
```
Comment (brikis98) [inline pages/careers/_open-positions.html]: How about:

* You know how to write code ("Dev")
* You have experience running production software ("Ops")

File: pages/careers/_open-positions.html [hunk 4]
```diff
@@ -1,8 +1,28 @@
...
+      <li>Bonus points for visible open source work.</li>
+      <li>Bonus points for any technical presentations you've done in the past.</li>
+      <li>Bonus points for a blog.</li>
+      <li>Bonus points for something in your profile that makes us smile.</li>
```
Comment (brikis98) [inline pages/careers/_open-positions.html]: What do you see as the benefit of listing these?

File: pages/careers/_open-positions.html [hunk 5]
```diff
@@ -1,8 +1,28 @@
...
+      <li>To be clear, you don't have to be good at everything. Our most important criteria are:
+        <ul>
+          <li>Strong cultural fit</li>
+          <li>You communicate well.</li>
+          <li>You have a broad and strong technical foundation.</li>
+          <li>You have a passion for writing software and a track record that shows it.</li>
```
Comment (brikis98) [inline pages/careers/_open-positions.html]: Again, what do you see as the benefit of writing this? Who is going to read this and say "oh dear, I don't communicate well, so I probably shouldn't apply."

File: pages/careers/_human-friendly.html [hunk 6]
```diff
@@ -1,28 +1,36 @@
...
-      <li>We have a minimum vacation policy.</li>
-      <li>Even though we are a 100% bootstrapped startup, we not only offer equity, but also pay market salaries.</li>
-      <li>Instead of having you spend a day solving artificial coding problems on a whiteboard as an interview, if we think you could be a good fit, we'll hire you for a day (or longer, if you're available) to work on a real project, using your normal coding environment, and we'll pay you for it.</li>
+      <li>We determine everyone's compensation using (non-public) formulas and we share those formulas with everyone in the company so you know that everyone is being treated exactly the same way. <a id="ref-1" href="#">[1]</a> <a id="ref-2" href="#">[2]</a></li>
+      <li>We have a minimum vacation policy. <a id="ref-3" href="#">[3]</a></li>
+      <li>We offer not only equity, but also pay above-market salaries, and offer a generous benefits package that includes 100% paid dental coverage, 100% paid vision coverage, and significant cost sharing on health coverage. <a id="ref-4" href="#">[4]</a> <a id="ref-5" href="#">[5]</a></li>
```
Comment (brikis98) [inline pages/careers/_human-friendly.html]: As per the comment below, just replace these with real anchor tags (`<a href="citation-1">`) and they will "just work."

Comment (josh-padnick) [reply pages/careers/_human-friendly.html]: Ah, I tried a few things, but didn't try that. I'll remove the code I wrote and use this simpler method.

Comment (josh-padnick) [reply pages/careers/_human-friendly.html]: 910bb5c76ad490434deb249764cbb9658a20c441

File: pages/careers/_open-positions.html (hunk 3 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html]: +1. Replaced.

File: pages/careers/_open-positions.html (hunk 4 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html]: These are things we actually value in a candidate. My thought is that by making it more clear what we value, people are better able to either self-identify as a good fit for present what we're looking to see.

Comment (josh-padnick) [reply pages/careers/_open-positions.html]: I cleaned these up a bit. 8bc6233bef92310eb9cf0b7fee1f219c5bf17875

File: pages/careers/_open-positions.html (hunk 5 above)
Comment (josh-padnick) [reply pages/careers/_open-positions.html]: I've seen great candidates disqualify themselves before so I wanted to make it apparent that we're not looking for perfection, just some foundational traits. That being said, to your point, "good communication" is definitely a job posting platitude. 

After thinking it over, I decided to remove these. Check out the latest updates in #37 and let me know what you think. I'll push the changes live but leave the PR open.

File: pages/careers/_scripts.html (hunk 1 above)
Comment (josh-padnick) [reply pages/careers/_scripts.html]: Per above, reverted.

**Pull request merged**